src/SuperstateToken.sol

```diff
@@ -61,21 +69,45 @@ abstract contract SuperstateToken is
     /// @notice Number of decimals used for the user representation of the token
     uint8 private constant DECIMALS = 6;
 
+    /// @notice Base 10000 for 0.01% precision
+    uint256 public constant FEE_DENOMINATOR = 10_000;
+
+    /// @notice Precision of SUPERSTATE_TOKEN
+    uint256 public constant SUPERSTATE_TOKEN_PRECISION = 10 ** DECIMALS;
+
+    /// @notice Lowest acceptable chainlink oracle price
+    uint256 public immutable MINIMUM_ACCEPTABLE_PRICE;
+
+    /// @notice Value, in seconds, that determines if chainlink data is too old
+    uint256 public maximumOracleDelay;
+
+    /// @notice The address of the oracle used to calculate the Net Asset Value per Share
+    address public superstateOracle;
+
+    /// @notice Mapping from a stablecoin's address to its configuration
+    mapping(address stablecoin => StablecoinConfig) public supportedStablecoins;
+
     /**
      * @dev This empty reserved space is put in place to allow future versions to add new fields without impacting
      * any contracts that inherit `SuperstateToken`
      */
-    uint256[100] private __additionalFieldsGap;
+    uint256[97] private __additionalFieldsGap;
```

Comment:
not sure if this is the first amongst many PRs, but we'll also want to update our storage tests to test these slots

Reply:
Will add to my list of follow ups for testing this